spaceship/lib/spaceship/stats_middleware.rb

```diff
@@ -18,11 +18,11 @@ def services
         @services ||= [
           ServiceOption.new("App Store Connect API (official)", "api.appstoreconnect.apple.com", "JWT"),
           ServiceOption.new("App Store Connect API (web session)", Spaceship::ConnectAPI::TestFlight::Client.hostname.gsub("https://", ""), "Web session"),
-          ServiceOption.new("App Store Connect API (web session)", Spaceship::ConnectAPI::Provisioning::Client.hostname.gsub("https://", ""), "Web session"),
           ServiceOption.new("Legacy iTunesConnect Auth", "idmsa.apple.com", "Web session"),
           ServiceOption.new("Legacy iTunesConnect Auth", "appstoreconnect.apple.com/olympus/v1/", "Web session"),
           ServiceOption.new("Legacy iTunesConnect", Spaceship::TunesClient.hostname.gsub("https://", ""), "Web session"),
-          ServiceOption.new("Legacy iTunesConnect Developer Portal", Spaceship::PortalClient.hostname.gsub("https://", ""), "Web session")
+          ServiceOption.new("Legacy iTunesConnect Developer Portal", Spaceship::PortalClient.hostname.gsub("https://", ""), "Web session"),
+          ServiceOption.new("App Store Connect API (web session)", Spaceship::ConnectAPI::Provisioning::Client.hostname.gsub("https://", ""), "Web session")
```

Comment:
why did you change the order in this?

Reply:
I think that the reason was that after the changes
`Spaceship::PortalClient.hostname` is `https://developer.apple.com/services-account/QH65B2`
`Spaceship::ConnectAPI::Provisioning::Client.hostname` is `https://developer.apple.com/services-account/` 
and without changing the order, all the calls would be matched by more generic `https://developer.apple.com/services-account/`